src/pages/resultsView/mutation/ResultsViewMutationMapper.tsx

```diff
@@ -92,6 +92,7 @@ export default class ResultsViewMutationMapper extends MutationMapper<IResultsVi
                 enableHotspot={this.props.config.show_hotspot}
                 enableMyCancerGenome={this.props.config.mycancergenome_show}
                 enableCivic={this.props.config.show_civic}
+                showAllExon={false}
```

Comment:
@leexgh 	no need to add this as a prop on resultsView. In other cases where we have a difference between the visualization of results and patient view, we hardcode in the patient view mutation table what this property should be (in the patient view table case `true`)

Reply:
see e.g.: https://github.com/cBioPortal/cbioportal-frontend/blob/master/src/pages/patientView/mutation/PatientViewMutationTable.tsx#L92-L93

and the one you added yourself: 

https://github.com/cBioPortal/cbioportal-frontend/blob/master/src/pages/patientView/mutation/PatientViewMutationTable.tsx#L112